### DISCUSSION TITLE
zh-cn/manual/cli/usage

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#157] [Comments]
# zh-cn/manual/cli/usage

使用 maa-cli 主要功能是通过调用 MaaCore，自动化完成明日方舟的游戏任务。此外，为了方便使用，maa-cli 还提供了管理 MaaCore 的功能。 管理 MaaCore maa-cli 可以安装和更新 MaaCore 及资源，只需运行以下命令： 更新 maa-cli 自身 maa-cli 可以更新自身，只需运行以下命令： 注意：使用包管...

https://docs.maa.plus/zh-cn/manual/cli/usage.html

**Comment by moonlighthger** [comment]
请问公招使用哪个命令？
